internal/xds/translator/fault.go

```diff
@@ -180,11 +180,20 @@ func (*fault) patchRoute(route *routev3.Route, irRoute *ir.HTTPRoute) error {
 	return nil
 }
 
-// translatePercentToFractionalPercent translates a v1alpha3 Percent instance
-// to an envoy.type.FractionalPercent instance.
+// TODO(liorlieberman) Below 2 functions should probably go somewhere else?
```

Comment:
can we combine these functions and move it into https://github.com/envoyproxy/gateway/blob/main/internal/xds/translator/utils.go ?

Reply:
moved. I think its cleaner to leave it as two functions.